suite-common/connect-init/src/connectInitThunks.ts

```diff
@@ -96,6 +96,7 @@ export const connectInitThunk = createThunk<void, ConnectInitHooks | void, void>
             switch (action.type) {
                 case UI.REQUEST_PIN:
                 case UI.INVALID_PIN:
+                case UI.REQUEST_PASSPHRASE:
```

Comment:
I am not sure whether I like the buttonRequest array and how it's handled. It's very loose implementation and doesn't have good & unified management across the repo (resetting the array for example). We can remove button requests and add them, but you have no guarantee of this logic at the place where you're selecting, which is where I prefer to react to the connect events rather than button requests selectors.